apps/emqx/test/emqx_takeover_SUITE.erl

```diff
@@ -884,26 +884,23 @@ t_kick_session(Config) ->
         {will_qos, 1}
     ],
     Commands =
-        %% GIVEN: client connect with willmsg payload <<"willpayload_kick">>
-        [{fun start_client/5, [ClientId, ClientId, ?QOS_1, WillOpts]}] ++
-            [
-                {fun start_client/5, [
-                    <<ClientId/binary, <<"_willsub">>/binary>>, WillTopic, ?QOS_1, []
-                ]}
-            ] ++
-            [
-                %% kick may fail (not found) without this delay
-                {
-                    fun(CTX) ->
-                        timer:sleep(100),
-                        CTX
-                    end,
-                    []
-                }
-            ] ++
+        lists:flatten([
```

Comment:
Nit. This list already looks flat.

Reply:
Oops, already merged.  :see_no_evil: 
I used the same format as the other test cases, so it would be uniform and simple to add a step to publish multiple messages, for instance.